cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/OutputsUtil.java

```diff
@@ -150,4 +151,11 @@ public static void putResponseIn(Map<String, String> queryMapResult, String outp
 
         return results;
     }
+
+    public static Map<String, String> getSuccessResultMapTerminateProvisionedProduct(TerminateProvisionedProductResult result) {
+        Map<String, String> results = new HashMap<>();
```

Comment:
OutputUtils is a generic class at project level while this method seems to be specific to Service Catalog module. Is it possible to pull it at lower level eg in servicecatalog package?